quickwit/quickwit-control-plane/src/control_plane.rs

```diff
@@ -66,9 +66,9 @@ use crate::IndexerPool;
 
 /// Interval between two controls (or checks) of the desired plan VS running plan.
 pub(crate) const CONTROL_PLAN_LOOP_INTERVAL: Duration = if cfg!(any(test, feature = "testsuite")) {
-    Duration::from_millis(500)
+    Duration::from_millis(100)
 } else {
-    Duration::from_secs(3)
+    Duration::from_secs(5)
```

Comment:
We now react to nodes joining and leaving the cluster, so I think it makes sense to improve that value now.